tests/test_utils/test_result.py

```
@@ -0,0 +1,120 @@
+        assert result is None
+
+    def test_default_zero_returns_zero(self):
+        result = unwrap_or({"error": "error"}, 0)
+        assert result == 0
+
+    def test_default_string_returns_string(self):
+        result = unwrap_or({"error": "error"}, "cached_value")
+        assert result == "cached_value"
+
+    def test_default_callable_not_evaluated_but_returned(self):
+        default = lambda: "not executed"
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `default` as a `def`

(E731)